syncano/models/options.py

```diff
@@ -95,7 +96,7 @@ def add_field(self, field):
             raise SyncanoValueError('Field "{0}" already defined'.format(field.name))
 
         self.field_names.append(field.name)
-        self.fields.append(field)
+        self.fields.insert(bisect(self.fields, field), field)
```

Comment:
I need this in CLI, it looks silly when u want to create a new instance and first field is description or in list view we should start from most important fields.